### PR TITLE
control-service: add logsUrl to DataJobExecution model

### PR DESCRIPTION
**Pull request by mivanov1988**
Integrated JobExecutionLogsUrlBuilder with DataJobExecution model.

Testing Done: unit and integration tests

Signed-off-by: Miroslav Ivanov miroslavi@vmware.com

### CODE REVIEW

File: projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java [hunk 1]
```diff
@@ -196,7 +198,7 @@ public void cancelDataJobExecution(String teamName, String jobName, String execu
 
       return dataJobExecutions
             .stream()
-            .map(dataJobExecution -> ToApiModelConverter.jobExecutionToConvert(dataJobExecution))
+            .map(dataJobExecution -> ToApiModelConverter.jobExecutionToConvert(dataJobExecution, jobExecutionLogsUrlBuilder.build(dataJobExecution)))
```
Comment (tozka) [inline projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java:201]: You probably need to update `getLastExecution` as well . 

You may consider some common helper method as well.

Comment (mivanov1988) [reply projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java:201]: Do you mean JobExecutionService.getLastExecution()?

Comment (mivanov1988) [reply projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java:201]: Common method that wraps `jobExecutionLogsUrlBuilder.build(dataJobExecution)` or `ToApiModelConverter.jobExecutionToConvert(dataJobExecution, jobExecutionLogsUrlBuilder.build(dataJobExecution))`?

Comment (mivanov1988) [reply projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java:201]: I was thinking about such a method (`ToApiModelConverter.jobExecutionToConvert(dataJobExecution, jobExecutionLogsUrlBuilder.build(dataJobExecution))`) but I didn't figure out where to put it. Do you have any suggestion?

Comment (tozka) [reply projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java:201]: > Do you mean JobExecutionService.getLastExecution()? If yes it uses DB entity not the model.

Oh right, They both have the same name and I glossed over the package name.

Comment (tozka) [reply projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/execution/JobExecutionService.java:201]: > I was thinking about such a method (ToApiModelConverter.jobExecutionToConvert(dataJobExecution, jobExecutionLogsUrlBuilder.build(dataJobExecution))) but I didn't figure out where to put it. Do you have any suggestion?

It's a nit really. I meant local private method

**Review by tozka — APPROVED** [review]

**Pull request merged**